infrastructure/templates/public-api/main.bicep

```diff
@@ -181,19 +181,23 @@ var legacyResourcePrefix = subscription
 
 var resourceNames = {
   existingResources: {
+    acr: 'eesacr'
+    acrResourceGroup: acrResourceGroupName
     adminApp: '${legacyResourcePrefix}-as-ees-admin'
+    alertsGroup: '${legacyResourcePrefix}-ag-ees-alertedusers'
     analyticsFileShare: '${commonResourcePrefix}-${abbreviations.fileShare}-anlyt'
     analyticsStorageAccount: '${replace(commonResourcePrefix, '-', '')}${abbreviations.storageStorageAccounts}anlyt'
-    publisherFunction: '${legacyResourcePrefix}-fa-ees-publisher'
-    keyVault: '${legacyResourcePrefix}-kv-ees-01'
-    vNet: '${legacyResourcePrefix}-vnet-ees'
-    alertsGroup: '${legacyResourcePrefix}-ag-ees-alertedusers'
-    acr: 'eesacr'
-    acrResourceGroup: acrResourceGroupName
-    // The Test Resource Group has broken from the naming convention of other environments for Core Storage
+    backupVault: {
+      vault: '${commonResourcePrefix}-${abbreviations.backupVaults}'
+      psqlFlexibleServerBackupPolicy: '${commonResourcePrefix}-psql-flexible-server-${abbreviations.backupVaultPolicies}'
+    }
     coreStorageAccount: subscription == 's101t01' || subscription == 's101p02'
       ? '${legacyResourcePrefix}storageeescore'
       : '${legacyResourcePrefix}saeescore'
+    keyVault: '${legacyResourcePrefix}-kv-ees-01'
+    logAnalyticsWorkspace: '${commonResourcePrefix}-log'
+    publisherFunction: '${legacyResourcePrefix}-fa-ees-publisher'
+    // The Test Resource Group has broken from the naming convention of other environments for Core Storage
```

Comment:
This comment has been moved and needs moving back to the line that it was referencing.